res/controllers/Denon-MC6000MK2.midi.xml

```diff
@@ -769,9 +769,27 @@
                     <normal/>
                 </options>
             </control>
+            <control>
+                <group>[Channel2]</group>
+                <key>DenonMC6000MK2.rightDeck2.loopOutButtonInput</key>
+                <status>0x92</status>
+                <midino>0x39</midino>
+                <options>
+                    <script-binding/>
+                </options>
+            </control>
```

Comment:
While we still have this annoying intermediate XML layer, I recommend keeping the mappings for button press and button release adjacent to each other in the code to make it easier to identify bugs like this.